docs/glsl-fuzz-walkthrough.md

```diff
@@ -858,11 +981,37 @@ Explanation:
   * If the worker fails to respond twice, assume the shader fails to render.
 * `--seed -136936935`
   * Random seed for the reduction algorithm.
-* `--server http://localhost:8080 --worker-name my-worker`
+* `--server http://localhost:8080 --worker my-worker`
   * `glsl-reduce` will, by default, run shader jobs locally, which is not well-tested.
 We specify a server and worker name to run the shader jobs on a worker that is connected
 to the server.
 
+
+### Performing a bad result reduction for a compute shader
+
+If you have a variant compute shader that produces a different SSBO result compared with its associated reference, you can reduce the problem as follows.
+
+```sh
+# Copy the .info.json file for the reference into your working directory
+cp /path/to/reference.info.json .
+# Make a copy of the template interestingness test for compute shader differences
+cp graphicsfuzz/python/interestingness/compute_interesting_diff.py .
+# Make the script executable
+chmod +x compute_interesting_diff.py
+
+# Edit 'REFERENCE_RESULTS', 'WORKER_NAME' and 'SERVER_URL' in 'compute_interesting_diff.py' to suit your needs
+
+# Run a reduction using 'compute_interesting_diff.py' to decide whether each reduced shader is interesting.  Results will be placed in 'someoutputdir'.  --preserve-semantics ensures that only semantics-preserving changes are made during reduction.
```

Comment:
This is a very long line shell comment line in a code section in markdown, which will not be auto-wrapped, so I suggest to wrap it to approx. 80 char width.

Reply:
Done.